mods/d2k/languages/rules/en.ftl

```diff
@@ -280,287 +280,287 @@ actor-deathhand =
+actor-concrete-b-name = Large Concrete Slab
 
 actor-construction-yard =
-   .description = Produces structures.
-   .encyclopedia = The Construction Yard is the foundation of any base built on Arrakis. Construction Yards produce a small amount of power and are required for the building of any new structures. Protect this structure! It is critical to the success of your base.
+   .description = Produces structures
+   .encyclopedia = The Construction Yard is the foundation of any base built on Arrakis. It produces a small amount of power and is required for building new structures. Protect this structure! It is critical to the success of your base.
 
-    Construction yards are fairly strong, but vulnerable in varying degrees to all types of weapons.
+    Construction yards are fairly strong, but vulnerable to all weapons to varying degrees
    .name = Construction Yard
 
 actor-wind-trap =
-   .description = Provides power for other structures.
+   .description = Supplies power to other structures
    .name = Wind Trap
-   .encyclopedia = Wind Traps provide power and water to an installation. Large, above-ground ducts funnel wind currents underground into massive turbines which power generators and humidity extractors.
+   .encyclopedia = Wind Traps supply power and water to an installation. Large, above-ground ducts funnel wind currents underground into massive turbines, which drive power generators and humidity extractors.
 
-    Wind Traps are vulnerable to most types of weapons.
+    Wind Traps are vulnerable to most weapons
 
 actor-barracks =
-   .description = Trains infantry.
+   .description = Trains infantry
    .name = Barracks
-   .encyclopedia = Barracks are required to produce and train light infantry units. Barracks can be upgraded for the production of more advanced infantry in later missions.
+   .encyclopedia = Barracks are required to produce and train light infantry units. They can be upgraded to train more advanced infantry in later missions.
 
-    Barracks are vulnerable to most types of weapons.
+    Barracks are vulnerable to most weapons
 
 actor-refinery =
-   .description = Harvesters unload Spice here for processing.
+   .description = Harvesters unload Spice here for processing
    .name = Spice Refinery
-   .encyclopedia = The Refinery is the basis of all Spice production on Dune. Harvesters transport mined Spice to the Refinery where it is converted into credits. Refined Spice is automatically distributed among the Silos and Refineries for storage. A refinery can store 2000 spice. A Spice Harvester is delivered via Carryall once a Refinery is built.
+   .encyclopedia = The Refinery is the basis of all Spice production on Dune. Harvesters transport mined Spice to the Refinery where it is converted into credits. Refined Spice is automatically distributed to Silos and Refineries for storage. Each refinery can store up to 2,000 units of spice. A Spice Harvester is delivered by Carryall as soon as a Refinery is built.
```

Comment:
Same thought r.e. are we happy to hardcode values such as "2,000 units of spice"